src/tools/gxpkg.ss

```diff
@@ -159,36 +157,36 @@
       ((new)
        (pkg-new .package .name .link))
       ((build)
-       (build-pkgs .pkg .?build-release .?build-optimized .?build-debug .?local))
+       (build-pkgs .pkg .?build-release .?build-optimized .?build-debug .?global))
       ((clean)
-       (clean-pkgs .pkg .?local))
+       (clean-pkgs .pkg .?global))
       ((deps)
-       (manage-deps .deps .?add .?install .?update .?remove))
+       (manage-deps .deps .?add .?install .?update .?remove .?global))
       ((link)
-       (link-pkg .pkg .src .?local))
+       (link-pkg .pkg .src .?global))
       ((unlink)
-       (unlink-pkgs .pkg .?force .?local))
+       (unlink-pkgs .pkg .?force .?global))
       ((install)
-       (install-pkgs .pkg .?local))
+       (install-pkgs .pkg .?global))
       ((uninstall)
-       (uninstall-pkgs .pkg .?force .?local))
+       (uninstall-pkgs .pkg .?force .?global))
       ((update)
-       (update-pkgs .pkg .?local))
+       (update-pkgs .pkg .?global))
       ((list)
-       (list-pkgs .?local))
+       (list-pkgs .?global))
       ((retag)
-       (retag-pkgs .?local))
+       (retag-pkgs .?global))
       ((search)
        (search-pkgs .keywords .directory .?as-list))
       ((dir)
-       (manage-dirs .directories .?add .?remove .?local))
+       (manage-dirs .directories .?add .?remove .?global))
       ((env)
        (env-exec .command .command-args)))))
 
 ;;; commands
 (def (env-exec command args)
-  (set-local-env!)
-  (set-local-path!)
+  (setup-local-pkg-env! #t)
```

Comment:
No global option here?

Reply:
no, env just does it -- its purpose is to get you in the local env.

Reply:
I guess we can have it take the flag.

Reply:
done.